src/visitor/operation_transform_visitor.rs

```diff
@@ -162,6 +163,14 @@ impl VisitMut for OperationTransformVisitor<'_> {
                 }
             }
 
+            Expr::Arrow(arrow) => {
+                let transform_result = ArrowTransform::to_dd_arrow_expr(arrow);
+                if transform_result.is_modified() {
+                    expr.map_with_mut(|e| transform_result.expr.unwrap_or(e));
+                    // do not update status yet
+                }
```

Comment:
why aren't we calling to the visitor with the children?

I miss some test checking that things like:
```
() => `a.b${c}`
() => {
    res = 'a.b' + c
}
...
```
are rewritten as expected.

Reply:
ok got it, we don't need to call to the children here, because we are creating new block, and it is `block_transform_visitor` which will visit the new block

Reply:
`BlockTransformVisitor` is visiting the arrow body block after mutating the arrow expression